ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-ref.yaml

```diff
@@ -23,6 +23,11 @@ ref:
     default: ""
   - name: JQ_VERSION
     default: ""
+  - name: OCP_VERSION
+    default: "4.15"
+  - name: OPENSHIFT_VERSION
+    default: "4.15"
+    documentation: The version of OpenShift to install, in MAJOR.MINOR format.
```

Comment:
One or the other... 
We should also update the job configuration so 4.15 is 4.15, 4.16 is 4.16 et cetra. 

see ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.16.yaml